bigquery/tests/unit/test_client.py

```diff
@@ -830,7 +830,7 @@ def test_update_table_w_schema_None(self):
                 'projectId': project,
                 'datasetId': dataset_id,
                 'tableId': table_id},
-            'schema': {'fields': []}
+            'schema': []
```

Comment:
This line should not have changed. If we are looking at an API response, this should either be None or there needs to be a dictionary with the `fields` key.

Reply:
Actually, I think this line should represent the full "before" schema.

Reply:
Fixed with https://github.com/GoogleCloudPlatform/google-cloud-python/pull/4076/commits/43611f45f4260ee45709201b0645fbab522f080b